### ISSUE TITLE
Save_id and file_list

### ISSUE THREAD

**Issue by protasisimone** [WebDevStudios/Custom-Metaboxes-and-Fields-for-WordPress#741]
Hi all,
can i use a save_id with file_list? I'm try to use this but doesn't work, I've need upload images and I wanna use this for the alt text, any suggestion? 
Thanks a lot for ur help.

**Comment by protasisimone** [comment]
No one can help me?

Thanks

**Comment by joshuadavidnelson** [comment]
This version of CMB isn't being supported, you should try [CMB2](https://github.com/WebDevStudios/CMB2).

**Comment by protasisimone** [comment]
Thank for ur reply, I've solved this issue with this:

https://pippinsplugins.com/retrieve-attachment-id-from-image-url/

I've extracted a URL with CMB and I use this snippet for catch the ID.
